src/main/java/frc/robot/commands/GetCenterNoteX.java

```diff
@@ -59,18 +59,19 @@ public GetCenterNoteX(Translation2d targetNote, DriveTrain driveTrain, NoteVisio
             // new WaitUntilCommand(intake::hasNote).withTimeout(INTAKE_EXTRA_WAIT_TIME),
             new WaitCommand(INTAKE_EXTRA_WAIT_TIME),
 
-            // turn off Shooter and intake
-            new InstantCommand(shooter::turnOffShooter),
-            new InstantCommand(intake::stop),
-            new InstantCommand(() -> shooter.setSpeakerShootMode(true)),
-
             // drive to shoot position, and spin up Shooter while going (after feeder stops)
             m_driveTrain.followPath(m_returnPath)
                 .deadlineWith(
-                    // new WaitUntilCommand(() -> (shooter.getFeederRpm() < Shooter.FEEDER_RPM_TOLERANCE)).withTimeout(1.0)
-                    new WaitCommand(1.0)
-                        .andThen(new ActiveSetShooter(shooter, shooterPivot, this::getShootValues))),
-            
+                    new WaitCommand(0.5)
```

Comment:
This change worked as expected, to allow the return path to start sooner, but also still spin the intake, to finish pickup while driving back to shoot.